pkg/driver/efs_watch_dog.go

```diff
@@ -248,7 +264,7 @@ func (w *execWatchdog) runLoop(stopCh <-chan struct{}) {
 	for {
 		select {
 		case <-stopCh:
-			klog.Info("stopping...")
+			//klog.Info("stopping...")
```

Comment:
Why comment this out?

Reply:
Ah this sneaked in when I was running tests locally, will revert in next commit.